mu-trees/addon/resolvers/glimmer-wrapper/index.js

```diff
@@ -82,7 +82,15 @@ const Resolver = GlobalsResolver.extend({
     this._glimmerResolver = new GlimmerResolver(this.config, this.glimmerModuleRegistry);
   },
 
-  normalize: null,
+  normalize: !DEBUG ? null : function(specifier) {
```

Comment:
Can you hoist this above? Something like:

```js
const normalize = !DEBUG ? null : function normalize(specifier) {
  // ...snip...
};

GlobalsResolver.extend({
  // ...snip...
  normalize,
});
```

Reply:
Done 👍